cms/djangoapps/contentstore/tests/test_admin.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests that verify that the admin view loads.
+
+This is not inside a django app because it is a global property of the system.
+"""
+import ddt
+from django.test import TestCase
+from django.urls import reverse
+from edx_toggles.toggles.testutils import override_waffle_flag
+
+from openedx.core.djangoapps.user_authn.config.waffle import REDIRECT_TO_LMS
+
+
+@ddt.ddt
+class TestAdminView(TestCase):
+    """
+    Tests of the admin view.
+    """
+    @override_waffle_flag(REDIRECT_TO_LMS, True)
+    @ddt.data(('/admin/', 3), ('/admin/login', 3), (reverse('admin:login'), 2))
+    @ddt.unpack
+    def test_admin_login_redirect(self, admin_url, index):
+        """Admin login will redirect towards the site login page."""
+        response = self.client.get(admin_url, follow=True)
+        self.assertIn('/login/edx-oauth2/?next=', response.redirect_chain[index][0])
```

Comment:
It feels a little fragile to have explicit index here -- this test might spuriously break if something else changes in the chain of login redirects. How about this?
```suggestion
        assert any('/login/edx-oauth2/?next=' in r[0] for r in redirect_chain)
```
Would this be too loose?